src/web/docusaurus/docs/contributing/database-maintenance.md

```diff
@@ -0,0 +1,59 @@
+---
+sidebar_position: 8
+---
+
+# Database Maintenance
+
+The following describes how to manually work with the Postgres database.
+
+## Backup (Manual)
+
+1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production]) and locate the `pg-backup-cron-job` container
+2. Click the **Console** button and select `bin/sh` in order to get a shell in the container
+3. Run a backup:
+
+```sh
+/etc/periodic/daily/create-backup
+```
+
+This will create a new dump file in `/var/opt/pg_dumps` with the current date and time.
+
+## Restore a Backup
+
+1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production]) and locate the `pg-backup-cron-job` container
```

Comment:
Is the `production` suppose to be a link? 
I end up getting this result when I ran on my machine and checked the `.md` on GitHub.
![image](https://user-images.githubusercontent.com/59105665/163464392-08dbd269-f214-412f-b4c9-043882a7783c.png)

Reply:
According to @JerryHue, this will work when we do it on Docusaurus.

Reply:
so running the Docusaurus on my machine will not work?